include/yaramod/types/expressions.h

```diff
@@ -1387,6 +1446,10 @@ class FunctionCallExpression : public IdExpression
 	const Expression::Ptr& getFunction() const { return _func; }
 	const std::vector<Expression::Ptr>& getArguments() const { return _args; }
 
+	//TODO: check if is correct:
+	virtual TokenIt getFirstTokenIt() const override { return _func->getFirstTokenIt();  /*return _symbolToken.value();*/ }
```

Comment:
What does the commented out code mean?

Reply:
It is old code which is now wrong. I removed it.